projects/demo/src/modules/components/tag/tag.template.html

```diff
@@ -1,4 +1,5 @@
 <tui-doc-page
+    deprecated
```

Comment:
Will it be deleted in separate PR ?

Reply:
yes, after replacing tags to chips in `multiselect` / `input-tag`. After new textfields, probably